movies.py

```diff
@@ -0,0 +1,82 @@
+"""Movie ticket process system and database"""
+
+MOVIE_PRICE = 5.0
+
+def enough_money(customer):
+    """Check if customer has adequate funds"""
+    balance = customer["cash"]
+    if balance < MOVIE_PRICE:
+        return False
+    return True
+
+def purchase_ticket(movie, customer):
+    """Performs the steps required in purchasing a ticket and returns altered customer and movie"""
+    assert enough_money(customer)
+    customer = charge(customer)
+    customer = dispense_ticket(movie, customer)
+    movie = remove_seat(movie)
+    return customer, movie
+
+def charge(customer):
+    """Charge customer the price of one ticket"""
+    assert enough_money(customer)
+    customer["cash"] = customer["cash"] - MOVIE_PRICE
+    return customer
```

Comment:
let's also condense down line 23 and 24 to just:
`return customer["cash"] - MOVIE_PRICE`